content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,149 @@
+to vendor neutrality and community-driven development enhancing the overall
+profiling and observability ecosystems.
+
+The donation happened through a great and constructive cooperation between
+Elastic and the OpenTelemetry community. We look forward to jointly establishing
+continuous profiling as an integral part of OpenTelemetry.
+
+With today’s acceptance, Elastic’s continuous profiling agent will be
+contributed to OpenTelemetry. This agent will now be jointly supported by both
+Elastic’s team as well as a diverse set of official maintainers from different
+companies:
+
+- Dmitry Filimonov (Grafana Labs)
+- Felix Geisendörfer (Datadog)
+- Jonathan Halliday (Red Hat)
+- Christos Kalkanis (Elastic)
+
+## What is continuous profiling?
+
+[Continuous profiling](https://www.cncf.io/blog/2022/05/31/what-is-continuous-profiling/)
+is a technique used to understand the behavior of a software application by
+collecting information about its execution over time. This includes tracking the
+duration of function calls, memory usage, CPU usage, and other system resources
+along with associated metadata.
+
+## Benefits of Continuous Profiling
+
+Traditional profiling solutions have significant drawbacks limiting adoption in
+production environments:
+
+- Significant cost and performance overhead due to code instrumentation
+- Disruptive service restarts
+- Inability to get visibility into third-party libraries
+
+Continuous profiling, however, runs in the background with minimal overhead,
+providing real-time, actionable insights without the need to replicate issues in
+separate environments.
+
+This allows SREs, DevOps, and developers to see how code affects performance and
+cost, making code and infrastructure improvements easier.
+
+## Contribution of comprehensive profiling abilities
+
+The continuous profiling agent, that Elastic is donating, is
+[based on eBPF](https://ebpf.io/) and by that a whole system, always-on solution
+that observes code and third-party libraries, kernel operations, and other code
+you don't own. It eliminates the need for code instrumentation
+(run-time/bytecode), recompilation, or service restarts with low overhead, low
+CPU (~1%), and memory usage in production environments.
+
+The donated profiling agent facilitates identifying non-optimal code paths,
+uncovering "unknown unknowns", and provides comprehensive visibility into the
+runtime behavior of all applications. The continuous profiling agent provides
+support for a wide range of runtimes and languages, such as:
+
+- C/C++
+- Rust
+- Zig
+- Go
+- Java
+- Python
+- Ruby
+- PHP
+- Node.js / V8
+- Perl
+- .NET
+
+Additionally, organizations can combine profiling data with resource information
+to derive an estimate for the carbon footprints of services. This helps
+organizations meet their sustainability objectives by minimizing computational
+wastage, ensuring seamless alignment with their strategic ESG goals.
```

Comment:
```suggestion
```

This feels redundant based on the fact that this is listed as a benefit below. 

As I wrote in one of my previous comments on that topic: I think we need to be wary with hype topics like sustainability, artifical intelligence, etc and not over-sell of them. I would rather play it low in this blog post and have a blog post in a year from now that gives a deep dive into how you can use profiling to do exactly that.